### PR TITLE
adding basic chart

### PR DESCRIPTION
**Pull request by notAnkur**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,24 +2,59 @@ import React, { Component } from 'react';
...
-  render() {
+
+    constructor() {
+        super();
+        this.state={
+            cDate: {}
+        }
+    }
+
+    componentWillMount() {
```
Comment (aneagoie) [inline src/App.js]: Let's change this to componentDidMount.

For this reason: https://stackoverflow.com/questions/46043832/why-componentwillmount-should-not-be-used

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "private": true,
   "homepage": "http://aneagoie.github.io/fullstacktrends",
   "dependencies": {
+    "chart.js": "^2.7.3",
```
Comment (aneagoie) [inline package.json:7]: do we need this or just `react-chartjs-2`?

**Review by aneagoie — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (notAnkur) [reply src/App.js]: will change that

File: package.json (hunk 2 above)
Comment (notAnkur) [reply package.json:7]: chartjs

**Review by aneagoie — APPROVED** [review]

**Comment by aneagoie** [comment]
Nice! Merging 👍

**Pull request merged**